internal/host/validator.go

```diff
@@ -870,3 +871,76 @@ func (v *validator) printDefaultRoute(c *validationContext, status ValidationSta
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *validator) isAPIDomainNameResolvedCorrectly(c *validationContext) ValidationStatus {
+	if !swag.BoolValue(c.cluster.UserManagedNetworking) {
```

Comment:
This validation is not enough. Single node will have this flag true.
Please create function that will check that this is None Platform multi node installation

swag.BoolValue(c.cluster.UserManagedNetworking) && !common.IsSingleNodeCluster(c.cluster)